mautic/mautic
# Focus item click conversion rate

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ N ]
| New feature/enhancement? (use the a.x branch)      | [ Y ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ Y ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR adds conversion rate stat to links in focus view and total views count in focus details.
![image](https://user-images.githubusercontent.com/8580942/221558963-c257dd83-6ae0-4412-aac1-416917df4709.png)



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Add focus item with **Emphasize a link** type and add some links to the content
3. Copy the focus installation script
4. Create new landing page and paste the focus installation script in the **Script inside head** field
5. Open the landing page using the link from **Page URL** and use private tab
6. Click the link from LP and generate some more statistics
7. Check if the statistics are correct
